src/main/java/com/fundynamic/d2tm/game/state/PlayingState.java

```diff
@@ -86,8 +90,9 @@ public void render(GameContainer container, StateBasedGame game, Graphics g) thr
 
     @Override
     public void update(GameContainer container, StateBasedGame game, int delta) throws SlickException {
+        float deltaF = delta / 1000f;
```

Comment:
Although I can make an educated guess, perhaps you can make this explicit that delta is (i think) in miliseconds? And what `deltaF` means?